Creating Remnant Crystal Research 1-6 (#5095)
endless-sky/endless-sky: data/remnant/remnant missions.txt

			label end
			`	"All right, enough history." He turns to the technicians who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
			scene "outfit/void rifle"

mission "Remnant: Crystal Research 1"
	name "Retrieve more Crystals"
	description `Retrieve Crystals from Greenwater, and then Allhome.`
	minor
	source "Viminal"
	destination "Greenwater"
	stopover "Allhome"
	cargo "Keystones" 2
	to offer
		random < 30
		has "Remnant: Learn Sign 2: done"
		has "Remnant: Technology Available: offered"
	on offer
		conversation

> 			`In the spaceport cafeteria you spot the outfitter who had you retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`

Reviewer: referring to the guy as "the outfitter" seems really weird to me

Author: Agreed. Changed.

			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down."`
			choice
				`	(Ask how business is going)`
				`	(Ask if there are more jobs like the last one you did for him.)`
					goto crystaljobs

> 			`	"Well, fairly well. Stocks are up, there are quite a few new outfits in the current, and some recent supply line disruptions have been resolved." He gestures appreciatively. "I am told that we have you to thank for some of that."`

Reviewer: thing after this leads to a quoted thingy, so needa remove the quotes at the end there

Author: Fixed. Thanks!

			`	He pauses to take a drink. "Unlike what you are probably used to from the outfitters you dealt with back in the ancestral worlds, my purpose is not to make money. Instead, my purpose is to ensure that Remnant captains have the resources they need. What you see listed as the 'price', Remnant captains see listed instead as a 'logistics adjustment' that indicates how much resources are needed to supply that component. Every ship has allocated resources and budgets determining how much they can utilize for repairs, restocking, and improvements."`
			choice
				`	"Wait, you didn't sell any of that?"`
				`	"OK, so that shipment you had me get?"`
					goto crystaljobs

> 			`	"You are correct. Despite what was said, none of it was sold - all of it was dismantled by researchers examining how it was made and what its properties are."`

Reviewer: same here

Author: Fixed.

			label crystaljobs
			`	"I have to apologize for that. We knew from our histories that anyone from the ancestral worlds would be attracted by the opportunity to earn a profit, and we needed to give you both a reason to stay, and a way to demonstrate your willingness to help. While that shipment was definitely useful, we had to frame it in a way that didn't reveal what we were doing, and give you an explanation that would make sense for doing it."`
			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
			choice
				`	(sign the affirmative)`

> 				`	(decline)`

Reviewer: the choices should have upper case for the first letter, and a period at the end, no?

Author: Partially fixed. I'm not sure if "Decline" needs a period, though.

			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have been combining them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`

mission "Remnant: Crystal Research 6"
	name "Deliver Data to <destination>"
	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
	source "Viminal"
	destination
		government "Remnant"
	to offer
		has "Remnant: Crystal Research 5: done"
	on offer
		conversation
			`You walk into the cafeteria and spot the outfitter halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have aclimitized to a few of the dishes, most of them are still very unappetizing.`
			choice

> 				`	"Remnant sure like to eat."`

Reviewer: this and the choice below it, as I understood it you were trying to go for something like "Why do Americans [do X]?", in referring to the Remnant as a sort of regional people, yes? I don't think it quite works like that though, dunno how to explain it but the word Remnant on both choices feel off, like something is missing to make them work and all. Guess an example would be if someone just said "British like to drink tea", which would seem odd compared to "The british like to drink tea", but I don't know if that'd help with the Remnant case. maybe "you Remnant" ? but then that might seem kinda douchey from <first> <last> I guess so idk.

Author: That's a good point. I don't think adding "The" in makes it any worse, and it does potentially make it better, so changed it.

				accept
	on stopover
		dialog
			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to help your passengers have the smoothest flight!" reads one of the labels.`
	on complete
		payment 250000
		dialog
			`You are just checking in with the port authorities to locate your shipment when your commlink indicates a pair of messages incoming. While you wait for the second one to load, you glance through the first one which is a simple deposit statement indicating that a payment of <payment> has been deposited from a "Solar Winds Logistics Ltd."`

mission "Remnant: Crystal Research 3"
	name "Retrieve more Crystals"
	description `Return the Crystals to Viminal.`
	landing
	source "Newhome"
	destination "Viminal"
	cargo "Keystones" 19
	to offer
		has "Remnant: Crystal Research 2: done"
	on offer
		dialog

> 			`The last few crates are loaded. Look over the assembled stack, you note that each crate bears the logos and advertisements for a different organization, shipper, or enterprise. The second message turns out to simply be a confirmation that you have picked up 19 Keystones. While there are no new instructions, the subtext is clear: Time to head back.`

Reviewer 2: ```suggestion
			`The last few crates are loaded. Looking over the assembled stack, you note that each crate bears the logos and advertisements for a different organization, shipper, or enterprise. The second message turns out to simply be a confirmation that you have picked up 19 Keystones. While there are no new instructions, the subtext is clear: Time to head back.`
```

		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

mission "Remnant: Crystal Research 4"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Keystone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling (Keystone)" "Winds of Light"
	
	to offer
		has "Remnant: Crystal Research 3: done"
	on offer
		conversation
			`You exit the ship to find that the pilot is actually Aeolus, and he has already marshalled a few dock workers with a number of wagons and lifters to unload the cargo.`
			`	"Ah, <first>, thank you for retrieving all these Keystones." he pauses to stare at a crate bearing the image of a bunch of smiling Hai, then shakes his head and turns back to you.`

> 			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here, swap out the Quantum Key Stone built with Hai crystals for one made from our crystals. Then we go back to Peragenor and get scanned again. By using the same ships for both tests, we should be able to get the best data."`

Reviewer 2: ```suggestion
			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here, swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we go back to Peragenor and get scanned again. By using the same ships for both tests, we should be able to get the best data."`
```

			choice
				`	"Wait, you are a pilot?"`
				`	"OK"`
					accept
			`	"Oh, yes. It is actually quite a common skill among the Remnant. I am not a particularly good pilot, but it is sufficiently important that it is taught to almost everyone as a general skill. Especially those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and depreciation as he talks about his piloting skills. "Anyway, let's go"`
	on visit
		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation

> 			`When you and the <npc> settle back onto the landing pad there is already a dockworker waiting with a Key Stone. Once the starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`

Reviewer 2: ```suggestion
			`When you and the <npc> settle back onto the landing pad there is already a dockworker waiting with a Key Stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
```

mission "Remnant: Crystal Research 5"
	name "Scan the <npc> in <waypoints>"

> 	description `Go scan the <npc> to record the Keystone post-wormhole.`

Reviewer 2: ```suggestion
	description `Go scan the <npc> to record the Key Stone post-wormhole.`
```

			label end
			`	"All right, enough history." He turns to the technicians who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
			scene "outfit/void rifle"

mission "Remnant: Crystal Research 1"
	name "Retrieve more Crystals"
	description `Retrieve Crystals from Greenwater, and then Allhome.`
	minor
	source "Viminal"
	destination "Greenwater"
	cargo "(reserved)" 2
	to offer
		random < 30
		has "Remnant: Learn Sign 2: done"
		has "Remnant: Technology Available: offered"
	on offer

> 		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to conventional outfitters in the rest of human space, he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`

Author: Changed it a little bit:

> In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.

> 		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to conventional outfitters in the rest of human space, he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`

Reviewer 3: Looks good (resolved).

		conversation
			`In the spaceport cafeteria you spot the outfitter who had you retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`

> 			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful. I never introduced myself before. My name is Aeolus. How can I help you today?"`

Author: I agree. Changed it around a bit. Now the latter part reads:

> "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "you came in here looking like you were watching for something. Anything on your mind?"

> 			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful. I never introduced myself before. My name is Aeolus. How can I help you today?"`

Reviewer 3: `You` needs to be capitalized in `ventures, "you came in here`. Besides that, nice rewording (resolved).

			choice
				`	(Ask how business is going)`
				`	(Ask if there are more jobs like the last one you did for him.)`
					goto crystaljobs
			`	"Well, fairly well. Stocks are up, there are quite a few new outfits in the current, and some recent supply line disruptions have been resolved." He gestures appreciatively. "I am told that we have you to thank for some of that.`
			`	"Those crystals you brought back have been quite interesting. The process of dismantling them for analysis yielded some interesting results." He notices your questioning expression. "Oh, you were expecting me to sell them, weren't you?"`
			choice
				`	(Nod affirmatively.)`
				`	(Shrug noncommittally.)`

> 			`	He looks slightly embarrassed. "Sorry to disappoint you, but none of those outfits were ever going to be used in the form they came in, for several different reasons." He ticks them off on his fingers as he lists them. "Firstly, security: Slapping an alien device on every ship without having thoroughly dismantled them would be an invitation to being tracked. Secondly, research: Something that does what the Quantum Key Stone does without being identical to it? Definitely interesting to our researchers. And thirdly, purpose."`

Author: "First" does sound better, so removed all the "ly" s from the numbers.

I'm not sure what you mean by the "being", though. I can only see one "being" there.

> 			`	He looks slightly embarrassed. "Sorry to disappoint you, but none of those outfits were ever going to be used in the form they came in, for several different reasons." He ticks them off on his fingers as he lists them. "Firstly, security: Slapping an alien device on every ship without having thoroughly dismantled them would be an invitation to being tracked. Secondly, research: Something that does what the Quantum Key Stone does without being identical to it? Definitely interesting to our researchers. And thirdly, purpose."`

Reviewer 3: I addressed this in my next review. Not sure why I was having trouble replying to this but it's fine now. Anyway, I was talking about `would be an invitation to being tracked`.

> 			`	He pauses to take a drink. "Unlike what you are probably used to from the outfitters you dealt with back in the ancestral worlds, my purpose is not to make money. Instead, my purpose is to ensure that Remnant captains have the resources they need. What you see listed as the 'price', Remnant captains see listed instead as a 'logistics adjustment' that indicates how much resources are needed to supply that component. Every ship has allocated resources and budgets determining how much they can utilize for repairs, restocking, and improvements."`

Author: Agreed, changed.

			choice
				`	"Wait, you didn't sell any of that?"`
				`	"OK, so that shipment you had me get?"`
					goto crystaljobs
			`	"You are correct. Despite what was said, none of it was sold - all of it was dismantled by researchers examining how it was made and what its properties are." He pauses to finish one plate before continuing.`
			label crystaljobs

> 			`	"I have to apologize for that. We knew from our histories that anyone from the ancestral worlds would be attracted by the opportunity to earn a profit, and we needed to give you both a reason to stay, and a way to demonstrate your willingness to help. While that shipment was definitely useful, we had to frame it in a way that didn't reveal what we were doing, and give you an explanation that would make sense for doing it.`

Author: Changed, on both counts.

			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
			choice
				`	(Sign the affirmative.)`
				`	(Decline.)`
					decline

> 			`	"Great. They would like you to pick up another 19 keystones. Two differences this time: Firstly, we would like you to pick them up from several different planets. And secondly, by the time you reach Hai space we will have made arrangements so the shipments will be already paid for and waiting for you. Pick up the first stones on Allhome, then the next set on Greenwater. I will have a list sent to you by the time you reach Greenwater."`

Author: Changed the "firstly"s.

There's no explicit lore concerning it, so I have expanded this section by giving the player a chance to ask about it. 

Note: There is a branch in here as well as a choice. If the player chooses to ask about it and has done espionage for the Deep, and triggered consequences that aren't in the game yet, they will have the "remnantuntrusted" attribute, in which case Aoleus won't be so forthcoming with his explanation.

> 			`	"That being said, if you happen to pick up any additional crystals, you can turn them into the logistics office. They will handle the distribution and compensate you accordingly."`

Author: Hmmm... I'm not sure how to change this one. Basically, Aeolus doubted the player would work to further their research, but would work out of greed.

> 			`	"That being said, if you happen to pick up any additional crystals, you can turn them into the logistics office. They will handle the distribution and compensate you accordingly."`

Reviewer 3: That makes sense. This sounds like it's more something I was confused about but might not confuse the majority of- if not all- other players. Probably fine to leave as is then (resolved).

				accept
	on complete
		dialog

> 			`It takes a few minutes to contact the yardmaster, but once you do the Hai have the crates moved from the warehouse to your bay doors fairly promptly. Hai and Humans seem accustomed to working together here.`

Author: Changed.

mission "Remnant: Crystal Research 2"
	name "Retrieve more Crystals"
	description `Retrieve Crystals from Greenwater, and then Allhome.`
	landing
	source "Greenwater"
	destination "Allhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 1: done"
	on complete
		payment 250000
		dialog

> 			`True to his word, there are crates of Hai quantum keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message from Viminal has arrived. It appears to be a notificatoin that a payment of <payment> has been made to your account, with a note attached that simply states "on-assignment allowance."`

Reviewer 4: `notificatoin` -> `notification`

Author: Thanks for spotting this. Fixed!

> 			`True to his word, there are crates of Hai quantum keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message from Viminal has arrived. It appears to be a notification that a payment of <payment> has been made to your account, with a note attached that simply states "on-assignment allowance."`

Author: Made the first change, and adjusted the end sentence by removing the "appears to be" part (replaced by "is a" ) so it is shorter and less wordy. 

How does that read now?

mission "Remnant: Crystal Research 3"
	name "Retrieve more Crystals"
	description `Retrieve Crystals from several Hai worlds, ending on Newhome.`
	landing
	source "Allhome"
	destination "Newhome"
	stopover "Makerplace"
	stopover "Stonebreak"
	stopover "Giverstone"
	stopover "Cloudfire"
	stopover "Snowfeather"
	stopover "Icelake"
	stopover "Heartvalley"
	cargo "Keystones" 4
	to offer
		has "Remnant: Crystal Research 2: done"
	on offer
		dialog

> 			`The message is brief and to the point. "Greetings, Captain <first>. Hopefully the shipments were waiting for you. We have arranged for samples to be acquired from Makerplace, Stonebreak, Giverstone, Cloudfire, Snowfeather, Icelake, and Heartvalley. Once you have collected all those, head to Newhome. There are a few other places that we are still working to setup, and we should have those ready by the time you reach Newhome."`

Author: Fixed. Thanks!

	on stopover
		dialog

> 			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to help your passengers have the smoothest flight!" reads one of the labels.`

Author: It should only display once. That being said, I'm going to replay through this to ensure it isn't doubling up on one of the messages.

> 			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to help your passengers have the smoothest flight!" reads one of the labels.`

Reviewer 3: (Resolved.)

	on complete
		payment 250000
		dialog

> 			`You are just checking in with the port authorities to locate your shipment when your commlink indicates a pair of messages incoming. While you wait for the second one to load, you glance through the first one which is a simple deposit statement indicating that a payment of <payment> has been deposited from a "Solar Winds Logistics Ltd."`

Reviewer 3: Hasn't been addressed yet. Adding a comment to make sure you see it.

mission "Remnant: Crystal Research 3A"
	invisible
	landing
	source "Makerplace"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"
	on offer

> 		dialog `Sure enough, there is a pair of crates waiting for you. This one features pictures of a pair of elderly and you suppose probably distinguished looking Hai in a fairly generic scene.`

Author: Changed.

mission "Remnant: Crystal Research 3C"
	invisible
	landing
	source "Giverstone"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3D"
	invisible
	landing
	source "Cloudfire"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3E"
	invisible
	landing
	source "Snowfeather"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3F"
	invisible
	landing
	source "Icelake"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3G"
	invisible
	landing
	source "Heartvalley"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 4"
	name "Retrieve more Crystals"
	description `Return the Crystals to Viminal.`
	landing
	source "Newhome"
	destination "Viminal"
	cargo "Keystones" 19
	to offer
		has "Remnant: Crystal Research 3: done"
	on offer
		dialog

> 			`The last few crates are loaded. Looking over the assembled stack, you note that each crate bears the logos and advertisements for a different organization, shipper, or enterprise. The second message turns out to simply be a confirmation that you have picked up 19 Keystones. While there are no new instructions, the subtext is clear: Time to head back.`

Author: Removed.

	on complete
		payment 2000000
		conversation

> 			`As you descend through the clouds to Viminal you get the feeling someone is watching you, but your sensors keep showing nothing more than a smattering of gulls and a patrol squad in the distance. Looking down, you spot an extra battery of anti-air artillery next to the main entrance to the spaceport. You note the artillery tracking you and quickly tap in the recognition code to the IFF system. Within seconds, the large guns are swinging back to covering the initial approach.`

Author: Changed.

> 			`	Passing through the cavernous opening into the great spaceport dome, there is a shimmer in the air next to you and a starling fades into view, settling onto the platform beside you. The pilot signs a quick "Greetings, anticipating talking with you." before heading back toward their lock. Behind them, the Starling continues its shutdown checks without any supervision at all.`

Author: I think I'll leave it as "them." I personally imagine them wearing a bunch of flight gear or otherwise being difficult to tell at a distance.

> 			`	As you finish your own shutdown routine your commlink informs you that your account has been updated with <payment>.`

Author: Fixed.

			"drag" 3.8
			"heat dissipation" 0.7
			"fuel capacity" 900
			"ramscoop" 1
			"cargo space" 36
			"outfit space" 312
			"weapon capacity" 83
			"engine capacity" 94
			"shield generation" 1.8
			"shield energy" 1.2
			"hull repair rate" 0.8
			"hull energy" 0.6
			"cloak" .02
			"cloaking energy" 5
			"cloaking fuel" .1
			"outfit scan power" 12
			"outfit scan speed" 1
			"tactical scan power" 26
			weapon
				"blast radius" 80
				"shield damage" 800
				"hull damage" 400
				"hit force" 1200
		outfits
			"Inhibitor Cannon" 3
			"Point Defense Turret"
			
			"Epoch Cell"
			"Crystal Capacitor"
			"Emergency Ramscoop" 2
			"Quantum Keystone"
			"Salvage Scanner"
			
			"Forge-Class Thruster"
			"Forge-Class Steering"
			"Crucible-Class Steering"
			"Hyperdrive"

		engine -16 82 0.7
		engine 16 82 0.7
		engine 0 90 1
		gun 0 -99 "Inhibitor Cannon"
		gun -12 -72 "Inhibitor Cannon"
		gun 12 -72 "Inhibitor Cannon"
		gun -18 -49
		gun 18 -49
		turret 0 2 "Point Defense Turret"
		leak "remnant leak" 30 4
		leak "remnant leak sparkle" 30 10
		explode "tiny explosion" 40
		explode "small explosion" 20
		explode "medium explosion" 10
		"final explode" "final explosion small" 1

> 		description "One of the strangest and most useful things that the Remnant discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They designed their scout ships to be able to cloak when needed, in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."

Reviewer 3: Change `discovered` to `has discovered` for grammatical consistency. Also, change `they designed` to `they've designed`/`they have dssigned`, then change `to be able to`to just `to` and remove the comma.

Author: Changed, although I used "have" instead of "has" at the start: `have discovered.` Because "has" sounded too much like I was talking about someone named "Remnant" as opposed to a group.

		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

mission "Remnant: Crystal Research 5"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Keystone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling (Keystone)" "Winds of Light"
	to offer
		has "Remnant: Crystal Research 4: done"
	on offer
		conversation

> 			`You exit the ship to find that the pilot is actually Aeolus, and he has already marshalled a few dock workers with a number of wagons and lifters to unload the cargo.`

Reviewer 3: `gathered` might be a better choice than `marshalled`. But that's subjective: if you like it, keep it.

Author: I like marshalled. It has a more organized feel to it than gathered does.

			`	"Ah, <first>, thank you for retrieving all these Keystones." he pauses to stare at a crate bearing the image of a bunch of smiling Hai, then shakes his head and turns back to you.`

> 			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here, swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we go back to Peragenor and get scanned again. By using the same ships for both tests, we should be able to get the best data."`

Reviewer 3: Replace the comma after `then we will return here` with an `and`. Also, `Then we go back to` should be `Then we will go back to` (I think? Either might be fine. I'm on the fence about `Then we go`, but I'm positive `Then we will go` is grammatically correct).

Author: First comma replaced. Added a "will" in the later one. So should be better now.

			choice
				`	"Wait, you are a pilot?"`
				`	"OK"`
					accept

> 			`	"Oh, yes. It is actually quite a common skill among the Remnant. I am not a particularly good pilot, but it is sufficiently important that it is taught to almost everyone as a general skill. Especially those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and depreciation as he talks about his piloting skills. "Anyway, let's go"`

Reviewer 3: `sufficiently` could he removed here. It took me a second to realize what was being explained because of that word usage. Just `important` might be better, but again, it's subjective. The second sentence (starts with `Especially`) is a fragment. Might I suggest: `It's especially worthwhile for those of us who work directly with ships on a daily basis and could be called upon to move or fly at a moment's notice.` Is `depreciation` a typo, or intentional? If intentional, you may want to consider a different word, as depreciation is more often associated with diminishing value of a certain item over time than it is the opposite of appreciation.

Author: I changed the third sentence; and swapped out the "depreciation" for "humility".

New version:
> I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a general part of general education. It is especially worthwhile for those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and humility as he talks about his piloting skills. "Anyway, let's go"

Reviewer 3: Looks good! Just forgot a period at the end there.

				accept
	on visit
		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation

> 			`When you and the <npc> settle back onto the landing pad there is already a dockworker waiting with a Key Stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`

Reviewer 3: Comma after `pad`. Also, I don't think it's possible to land gracefully from anything several meters up onto hard floor. Maybe replace `several-meter` with `high`? Remnants are still humans, after all.

Author: Comma added.

The several-meter figure was intentionally picked as a height that is high enough that most people would be uncomfortable jumping down from that height, but not impossibly so. (The average athletic person *can* jump down from that height safely, but probably not comfortably.)

mission "Remnant: Crystal Research 6"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Key Stone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling" "Winds of Light"
	to offer
		has "Remnant: Crystal Research 5: done"
	on offer
		conversation

> 			`	"Ready? Let's go!" signs Aeolus as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear of the ship and watches as the two ships head back out of the dome.`

Reviewer 3: The exclamation point after `Let's go` should be a comma, I believe.
https://www.grammarbook.com/punctuation/quotes.asp
Not so sure about this myself (Rule 3b is the closest match I can find to this usage), but my instincts are telling me it should be a comma.
Also, I'd remove `of the ship` to avoid using `ship` twice so close together.

Author: I like the emphasis that the exclamation point adds, so if you are unsure, let's get a second opinion on that part.

Removed the `of the ship` part. That does make it flow better.

Author: Discord discussion and input from TheMarksman agree with keeping the exclamation point here.

				launch
	on visit
		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation
			`The scanning proceeds smoothly, and while it does note some differences between the two crystals, they both appear to function similarly. After you return to Viminal, Aeolus comes over to collect a copy of the data for the researchers.`

> 			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully "Perhaps more importantly, it may help understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aids."`

Reviewer 3: Add a colon after `thoughtfully`. Add `us` after `help`. `aids` might sound better singular: `aid`.

Author: Changed.

> 			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`

Reviewer 3: `May the Embers burn bright for you`. Don't know if you came up for this or if I forgot it was included in earlier Remnant content, but that line is beautiful. If you made it, nicely done.

Author: Yes, I made it myself. This isn't technically the first time I've used it, but the other time is in the Cognizance PR which hasn't been merged yet. So it might be the first time that the player sees it.

I was inspired by Tali (from Mass Effect)'s code for confirming their approach to the migrant fleet: "After time adrift among open stars, along tides of light and through shoals of dust, I will return to where I began"

It is one of many little tidbits that I'm working on developing to make the Remnant a more unique and identifiable culture, and not just "strange humans that sign and sign." And the pace of the story makes sense for it, too: The player is gradually becoming more familiar with the Remnant, and more familiar to them. So they are opening up, including them, and treating them more as 'one of them.'

Reviewer 3: Good stuff. I can tell you're putting a lot of thought into this (resolved).

> 			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have been combining them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`

Reviewer 3: `been combining` would sound better as `combined` in my opinion.
Sidenote: I love it whenever someone uses "however" midsentence instead of the beginning. Don't know if you read Strunk and White in HS/college, but that usage was like the one thing I took away from that book.

Author: Changed.

Thanks. I have a tendency to overuse "however", but I am glad that I used it in a way that is appreciated in this case.

mission "Remnant: Crystal Research 7"
	name "Deliver Data to <destination>"
	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
	source "Viminal"
	destination
		government "Remnant"
	to offer
		has "Remnant: Crystal Research 6: done"
	on offer
		conversation
			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have acclimatized to a few of the dishes, most of them are still very unappetizing.`
			choice
				`	"The Remnant sure like to eat."`
				`	"Why do the Remnant eat so much?"`
					goto foodvolume
				`	"Maybe I should import some spices."`
					goto spices

> 			`	Aeolus chuckles and makes an amused gesture. "An army lives and dies by its food! A poorly fed soldier is but a short step away from being a deserter or even a traitor. While we have had lean years in the past, these days we ensure everyone is well fed."`

Reviewer 3: Add a comma or dash after `deserter`.

Author: Added a comma.

				goto deliverdata
			label foodvolume
			branch capitallicense
				"license: Remnant Capital" = 1
			`	He looks at you appraisingly. "You are perceptive, but there are some things you cannot learn yet. Ask us again when you have earned more of our trust."`
				goto deliverdata
			label capitallicense
			apply "remnant symbionts info" += 1

> 			`	He looks at you appraisingly "You are perceptive, and have earned our trust." He leans forward and holds up his hands. He pokes at the leather fingerless gloves he is wearing, and you note that they actually extend up under his sleeves. Looking closer, you note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`

Reviewer 3: I would switch `note` to `realize` and remove the `that` that immediately follows it. Also, in the next sentence, I would remove `Looking closer, you note that` and just go straight to the "meat" of the sentence. Maybe remove the `actually in there as well`.
Double midsentence "however"? 1 million cool points for you, my friend.

Author: Changed.

> 			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones." He holds up his hands. "Have a knack for precision and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed on us and require regular care and attention. Normally, this causes us no hardship; but if we do not eat enough we will consume our reserves far faster than we normally would."`

Reviewer 3: `These ones` should have a comma, not a period. Additionally, I would change that next bit to `he explains while holding up his hands,` to more smoothly transition back into the dialogue. I'm not a fan of the back-to-back usage of precise, but I can't think of any easy way to fix it, so it's probably fine.
I would also suggest changing `on us` to `through us` to make the symbionts seem less parasitic and more mutually beneficial.

Author: Made the changes.

			choice
				`	"That is quite the revelation."`
				`	"Thanks for the explanation."`
				`	"Is that why some of the food tastes terrible?"`
					goto terribleflavor
				`	"Sounds creepy."`
					goto creepy

> 			`	The outfitter nods. "From what I have heard, humanity does not even seem to have much in the way of exoskeletons or cybernetics. Having living organisms we wear would certainly seem to be the realm of science-fiction." He contemplates his symbiont for a few seconds as he absent-mindedly eats more of his meal. "But education aside, there is work to do."`

Reviewer 3: I'm guessing `outfitter` is here from before you gave him a name? Probably needs to be switched to `Aeolus` unless you intended to keep it.

Author: Good catch. Changed.

				goto deliverdata
			label spices
			`	He shakes his head. "I will not discourage you doing so - and I am sure some might appreciate them - but most of us like our food the way it is." He chuckles, then continues. "But there is still work to do."`
				goto deliverdata
			label terribleflavor
			`	"I suppose it might. Our symbionts have different dietary requirements, and such unusual compounds in our food inevitably change the flavor. We are all long since used to it, but it would definitely be different for an outsider." He seems amused by this. "But enough of food - there is work to do!"`
				goto deliverdata
			label creepy

> 			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts to being useful leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`

Reviewer 3: Not this specifically, but about the entire above conversation: nice work making this section here branching. Having him specifically respond to questions and statements you say seems more natural. The only thing I would warn is if symbionts are going to play a role in future plot points, ensure they are either explained again then (because I believe the player can go through this without ever learning about symbionts) or alter it so the player has to learn about symbionts no matter what conversation options they pick.

Author: Thank you. 

This is intended to be something of a "sneak peek" for the player. It will be a plot point in the future, and before then they will have more direct opportunities to learn about it.

Reviewer 3: Good to hear (resolved).

			label deliverdata
			`	He pulls out a small metal briefcase and sets it on the table. "Another delivery mission for you, Captain. A short one, at that. We need you to take this case of data crystals with our preliminary data to our researchers on <destination>. This will allow them to combine the data we have collected with their most recent findings, and perhaps help us take a step forward towards understanding the nature of both these crystals and the wormholes. Will you do it?"`
			choice

> 				`	"OK"`

Reviewer 3: Needs a period. Maybe change to `Okay.`, not `OK.`

Author: Changed and fixed.

					accept

> 				`	"Not right now"`

Reviewer 3: Period here as well. Short stuff deserves punctuation as much as the big stuff does.

Author: Fixed.

					defer
	on complete
		payment 100000
		conversation

> 			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain." she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them...." she makes a convoluted gesture that you think translates as "knowing that you don't know."`

Reviewer 3: Consider changing `you think` to `(you think)` to better accommodate the "break" in description. Also, maybe `to` instead of `as`.

Author: Changed.

> 			`	"Now, thanks to your acquisitions, we have the opportunity to get some new insights. We are looking forward to learning more about these anomalies. May the Embers burn bright for you, Captain." With that, she turns and heads off towards a lab.`

Reviewer 3: `to get some` can be simplified to `for`.

Author: Changed.

	on offer
		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`
		conversation
			`In the spaceport cafeteria you spot the outfitter who had you retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`
			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "You came in here looking like you were watching for something. Anything on your mind?"`
			choice
				`	(Ask how business is going)`
				`	(Ask if there are more jobs like the last one you did for him.)`
					goto crystaljobs

> 			`	"Well, fairly well. Stocks are up, there are quite a few new outfits in the current, and some recent supply line disruptions have been resolved." He gestures appreciatively. "I am told that we have you to thank for some of that.`

Author: I meant current, as in river, or wind. So "outfits in the current" meaning pretty much the exact same thing as "products in the pipeline".

> 			`	"Well, fairly well. Stocks are up, there are quite a few new outfits in the current, and some recent supply line disruptions have been resolved." He gestures appreciatively. "I am told that we have you to thank for some of that.`

Author: Added a conversation choice where the player can ask about that. Aeolus provides an explanation. :)

			`	"Those crystals you brought back have been quite interesting. The process of dismantling them for analysis yielded some interesting results." He notices your questioning expression. "Oh, you were expecting me to sell them, weren't you?"`

> 			choice

Author: Good idea. I have added in a conversation option for this.

				`	(Nod affirmatively.)`
				`	(Shrug noncommittally.)`

> 			`	He looks slightly embarrassed. "Sorry to disappoint you, but none of those outfits were ever going to be used in the form they came in, for several different reasons." He ticks them off on his fingers as he lists them. "First, security: Slapping an alien device on every ship without having thoroughly dismantled them would be an invitation to being tracked. Second, research: Something that does what the Quantum Key Stone does without being identical to it? Definitely interesting to our researchers. And third, purpose."`

Reviewer 3: (Still learning with Github, couldn't figure out how to respond to your reply about the two "being"s, so this is my reply to that.)
`would be an invitation to being tracked`: `be` and `being` are used close together in this spot, which is why I suggested changing `being` to `getting`. Not a huge issue since they're only the same root word, not the exact same word.

			choice
				`	"Wait, you didn't sell any of that?"`
				`	"Okay, so that shipment you had me get?"`
					goto crystaljobs
			`	"You are correct. Despite what was said, none of it was sold - all of it was dismantled by researchers examining how it was made and what its properties are." He pauses to finish one plate before continuing.`
			label crystaljobs
			`	"I have to apologize for that. We knew from our histories that anyone from the ancestral worlds would be attracted by the opportunity to earn a profit, and we needed to give you both a reason to stay, and a way to guage your willingness to help. Your shipment was definitely useful, but we had to ensure we framed your job in a way that made sense without revealing what we were actually doing.`
			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
			choice

> 				`	(Sign the affirmative.)`

Reviewer 5: ```suggestion
				`	(Sign in the affirmative.)`
```

			choice
				`	"Wait, how will you get this all set up?"`
				`	"Sounds good."`
					accept
			branch remnantuntrusted
				"remnantuntrusted" >= 1 # This attribute is currently not used, but will be set by some of the "consequence" missions as a result of spying for the Deep.
			`	Aeolus looks faintly amused "While you are certainly adventurous, you are not the only one to have traveled widely. We mapped the Hai systems long ago with cloaked ships, but never landed or contacted any of them. So we never knew anything more than what we could observe from orbit or from scanning ships. But you are known to them, and have a fairly public reputation. By using your transponder codes, we can place and pay for orders through hyperspace relays. All we need is a ship in position to feed the signal into the network.`
				accept
			label remnantuntrusted
			`	Aeolus smiles mysteriously at you. "Let's just say that you are not the only resourceful person working for us. We will share more with you once you have demonstrated your loyalty."`
				accept
	on complete
		dialog

> 			`It takes a few minutes to contact the yardmaster, but once you do the Hai have the crates moved promptly from the warehouse to your bay doors. Hai and Humans seem accustomed to working together here.`

Reviewer 5: ```suggestion
			`It takes a few minutes to contact the yardmaster, but once you, a mixed crew of Hai and Human dockworkers has the crates moved promptly from the warehouse through your cargo bay doors. Hai and Humans seem accustomed to working together here.`
```

Author: That mostly looks like an improvement, except for a missing "do,"

`but once you, a mixed crew of` -> `but once you do, a mixed crew of`

Author: OK, I incorporated your suggestion. Thanks!

mission "Remnant: Crystal Research 2"
	name "Retrieve more Crystals"
	description `Retrieve Crystals from Greenwater, and then Allhome.`
	landing
	source "Greenwater"
	destination "Allhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 1: done"
	on complete
		payment 250000
		dialog

> 			`True to Aeolus' word, there are crates of Hai Quantum Keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message from Viminal has arrived. It is a notification that a payment of <payment> has been made to your account, with a note attached that simply states "on-assignment allowance."`

Reviewer 5: It's been established that instantaneous communication relies on hyperspace relays, but Vimnal is not connected to Allhome via a direct hyperspace route, so an instant message from Vimnal shouldn't be possible. Maybe rephrase this to make it clear that the message is pre-recorded, or something like that.

Author: I was thinking a text message. Literally like how a credit card statement has "payee: amazon.com" or some such written next to the line. "payee: on-assignment allowance." I'm actually kind of hoping that the player notices that very discrepancy, and wonders why that is the case. (behind the scenes, I'm explaining it to myself as either 

a) the Remnant planted a device and/or code on the player's ship to activate and make the payment when conditions are met; or 
b) the Remnant are following the player in a cloaked ship and are transmitting the payment when they can tell the player has made the pickup.

Author: OK, I edited out the mention of viminal, and it now emphasizes that it doesn't mention where it is from. So now it is clear that this is not a normal message that, despite appearing on your commlink, is clearly irregular.

Reviewer 5: LGTM!

mission "Remnant: Crystal Research 3"
	name "Retrieve more Crystals"
	description `Retrieve Crystals from several Hai worlds, ending on Newhome.`
	landing
	source "Allhome"
	destination "Newhome"
	stopover "Makerplace"
	stopover "Stonebreak"
	stopover "Giverstone"
	stopover "Cloudfire"
	stopover "Snowfeather"
	stopover "Icelake"
	stopover "Heartvalley"
	cargo "Keystones" 4
	to offer
		has "Remnant: Crystal Research 2: done"
	on offer
		dialog
			`The message is brief and to the point. "Greetings, Captain <first>. Hopefully the shipments were waiting for you. We have arranged for samples to be acquired from Makerplace, Stonebreak, Giverstone, Cloudfire, Snowfeather, Icelake, and Heartvalley. Once you have collected all those, head to Newhome. There are a few other places that we are still working to set up, and we should have those ready by the time you reach Newhome."`
	on stopover
		dialog

> 			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to help your passengers have the smoothest flight!" reads one of the labels.`

Reviewer 5: ```suggestion
			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to provide the smoothest flight for your passengers!" reads one of the labels.`
```

	on complete
		payment 250000
		dialog
			`You are just checking in with the port authorities to locate your shipment when your commlink indicates a pair of messages incoming. While you wait for the second one to load, you see the first message is a deposit statement indicating that a payment of <payment> has been deposited from a "Solar Winds Logistics Ltd."`

mission "Remnant: Crystal Research 3A"
	invisible
	landing
	source "Makerplace"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"
	on offer

> 		dialog `Sure enough, there is a pair of crates waiting for you. This one features pictures of a pair of elderly and distinguished looking Hai in a generic scene.`

Reviewer 5: ```suggestion
		dialog `Sure enough, there is a pair of crates waiting for you. This one depicts of a pair of distinguished elderly Hai relaxing in front of a matte painting.`
```

		leak "remnant leak" 30 4
		leak "remnant leak sparkle" 30 10
		explode "tiny explosion" 40
		explode "small explosion" 20
		explode "medium explosion" 10
		"final explode" "final explosion small" 1
		description "One of the strangest and most useful things that the Remnant have discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They have designed their scout ships to cloak when needed in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."
		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

mission "Remnant: Crystal Research 5"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Keystone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling (Keystone)" "Winds of Light"
	to offer
		has "Remnant: Crystal Research 4: done"
	on offer
		conversation

> 			`You exit the ship to find that the pilot is actually Aeolus, and he has already marshalled a few dock workers with a number of wagons and lifters to unload the cargo.`

Reviewer 5: ```suggestion
			`You exit the ship to find that the pilot is actually Aeolus, who has already marshaled some dock workers with a number of wagons and lifters to unload the cargo.`
```

			`	"Ah, <first>, thank you for retrieving all these Keystones." he pauses to stare at a crate bearing the image of a bunch of smiling Hai, then shakes his head and turns back to you.`

> 			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we will go back to Peragenor and get scanned again. By using the same ships for both tests, we should be able to get the best data."`

Reviewer 5: ```suggestion
			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we will go back to Peragenor and get perform another scan. By using the same ships for both tests, we should be able to get the best data."`
```

Author: It looks like your suggestion replaces `get` with `perform`, but the `get` isn't actually deleted.

Author: OK, I've incorporated this suggestion. Thanks!

			choice
				`	"Wait, you are a pilot?"`
				`	"Okay."`
					accept

> 			`	"Oh, yes. It is actually quite common among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a part of general education. It is especially worthwhile for those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and humility as he talks about his piloting skills. "Anyway, let's go!"`

Reviewer 5: ```suggestion
			`	"Oh, yes. It is actually quite common among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a part of a general education. It is especially worthwhile for those of us who work directly with starships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures suggest both pride and humility as he talks about his piloting skills. "Anyway, let's go!"`
```

mission "Remnant: Crystal Research 6"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Key Stone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling" "Winds of Light"
	to offer
		has "Remnant: Crystal Research 5: done"
	on offer
		conversation
			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear and watches as the two ships head back out of the dome.`
				launch
	on visit
		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation

> 			`The scanning proceeds smoothly, and while it does note some differences between the two crystals, they both appear to function similarly. After you return to Viminal, Aeolus comes over to collect a copy of the data for the researchers.`

Reviewer 5: ```suggestion
			`The scanning has proceeded smoothly. While in transit, you took the opportunity to examine the preliminary results: a few differences between the two crystals, but both appear to have functioned similarly. Upon landing on Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
```

			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`
			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have combined them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`

mission "Remnant: Crystal Research 7"
	name "Deliver Data to <destination>"
	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
	source "Viminal"
	destination
		government "Remnant"
	to offer
		has "Remnant: Crystal Research 6: done"
	on offer
		conversation

> 			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have acclimatized to a few of the dishes, most of them are still very unappetizing.`

Reviewer 5: ```suggestion
			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have become accustomed to a few of the dishes, most of them are still very unappetizing.`
```

> 			choice

Reviewer 5: Maybe add a choice here that doesn't involve somehow commenting on the food.

Author: I have now added a "Straight to business" option to the conversation. It isn't nearly as interesting as the other options, but it is a valid option that I'm sure some players will prefer to take.

				goto deliverdata
			label spices
			`	He shakes his head. "I will not discourage you doing so - and I am sure some might appreciate them - but most of us like our food the way it is." He chuckles, then continues. "But there is still work to do."`
				goto deliverdata
			label terribleflavor
			`	"I suppose it might. Our symbionts have different dietary requirements, and such unusual compounds in our food inevitably change the flavor. We are all long since used to it, but it would definitely be different for an outsider." He seems amused by this. "But enough of food - there is work to do!"`
				goto deliverdata
			label creepy
			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts to being useful leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`
			label deliverdata

> 			`	He pulls out a small metal briefcase and sets it on the table. "Another delivery mission for you, Captain. A short one, at that. We need you to take this case of data crystals with our preliminary data to our researchers on <destination>. This will allow them to combine the data we have collected with their most recent findings, and perhaps help us take a step forward towards understanding the nature of both these crystals and the wormholes. Will you do it?"`

Reviewer 5: ```suggestion
			`	He pulls out a small metal briefcase and sets it on the table. "Another delivery mission for you, Captain. A short one, at that. We need you to take this case of data crystals with our preliminary data to some researchers on <destination>. This will allow them to combine the data we have collected with their most recent findings, and perhaps help us take a step forward towards understanding the nature of both these crystals and the wormholes. Will you do it?"`
```

			choice
				`	"Okay."`
					accept
				`	"Not right now."`
					defer
	on complete
		payment 100000
		conversation

> 			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain." she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them...." she makes a convoluted gesture that (you think) translates to "knowing that you don't know."`

Reviewer 5: ```suggestion
			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain." she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them...." she makes a convoluted gesture that you think translates to "knowing that you don't know."`
```

			"tactical scan power" 26
			weapon
				"blast radius" 80
				"shield damage" 800
				"hull damage" 400
				"hit force" 1200
		outfits
			"Inhibitor Cannon" 3
			"Point Defense Turret"
			
			"Epoch Cell"
			"Crystal Capacitor"
			"Emergency Ramscoop" 2
			"Quantum Keystone"
			"Salvage Scanner"
			
			"Forge-Class Thruster"
			"Forge-Class Steering"
			"Crucible-Class Steering"
			"Hyperdrive"

		engine -16 82 0.7
		engine 16 82 0.7
		engine 0 90 1
		gun 0 -99 "Inhibitor Cannon"
		gun -12 -72 "Inhibitor Cannon"
		gun 12 -72 "Inhibitor Cannon"
		gun -18 -49
		gun 18 -49
		turret 0 2 "Point Defense Turret"
		leak "remnant leak" 30 4
		leak "remnant leak sparkle" 30 10
		explode "tiny explosion" 40
		explode "small explosion" 20
		explode "medium explosion" 10
		"final explode" "final explosion small" 1
		description "One of the strangest and most useful things that the Remnant have discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They have designed their scout ships to cloak when needed in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."

> 		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

Author: Yes, I should shorten that up a bit.

That being said, no, I have no plans on having any Remnant ships with Hai Keystones be a thing. Quantum Keystones are an interesting thing to the Remnant - but despite what they say, the Keystones only real value is for research purposes. And, as described in the earlier mission dialog, they don't trust them. There's massive security implications in using outfits they don't understand - especially if manufactured deliberately for sale by another faction. 

Aside from testing purposes in this mission, there's no reason to use the Hai Keystone on any other Remnant ship. I suppose to be thorough I could add a follow up mission where we do this exact same experiment with every ship in the Remnant aresenal to see how they each react. (Scientifically, this would be the correct thing to do), but I think it is a case of gameplay vs realism at this point. But beyond this, the Keystone equipped ship won't be used further.

> 		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

Author: Upon making the change and doing some testing; no, I'm not going to shorten it up right now. For some reason, when it is shortened up into the condensed variant format, the NPC loses its "staying" behaviour. I prefer longer-than-strictly-necessary but working over short but not-working.

> 		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

Reviewer 6: The issue with the ship leaving the system appears to the tied to the variety of personalities used for the NPC, rather than with the definition of the ship (I was able to make it not jump away by replacing the personalities for the ship with the personalities of the Cruiser from FW Reconnaissance). What's noticeable is that the only reason that the ship doesn't leave the system is that it gets locked into a loop of trying to land on one of the planets (through some testing, I was occasionally able to break this behaviour by attacking it, then cloaking, making it jump).

	to offer
		random < 30
		has "Remnant: Learn Sign 2: done"
		has "Remnant: Technology Available: offered"
	on offer
		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`
		conversation
			`In the spaceport cafeteria you spot the outfitter who asked you to retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`
			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "You came in here with the appearance of looking for something. Is there something on your mind?"`
			choice
				`	(Ask how business is going.)`

> 				`	(Ask if there are more jobs like the last one you did for him.)`

Author: While I don't have a hard rule on when to use brackets and when to do specific replies, I prefer to use brackets for general statements like these so that the player can imagine their own preferred method of talking. Specific replies are best suited to when they include (or are responding to) information that <first> could be reasonably assumed to know but the player may have forgotten.

				accept
	on stopover
		dialog
			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to provide the smoothest flight for your passengers!" reads one of the labels.`
	on complete
		payment 250000
		dialog
			`You are just checking in with the port authorities to locate your shipment when your commlink indicates a pair of messages incoming. While you wait for the second one to load, you see the first message is a deposit statement indicating that a payment of <payment> has been deposited from a "Solar Winds Logistics Ltd."`

mission "Remnant: Crystal Research 3A"
	invisible
	landing
	source "Makerplace"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"
	on offer

> 		dialog `Sure enough, there is a pair of crates waiting for you. This one depicts of a pair of distinguished elderly Hai relaxing in front of a matte painting.`

Author: The silent missions are actually just there to add detail to the large "visit all these worlds" mission, so that the player can see the cargo accumulating in their hold if they check.

> 		dialog `Sure enough, there is a pair of crates waiting for you. This one depicts of a pair of distinguished elderly Hai relaxing in front of a matte painting.`

Author: Changed it from a dialog to a conversation, thus forcing the accept.

			"Crucible-Class Steering"
			"Hyperdrive"

		engine -16 82 0.7
		engine 16 82 0.7
		engine 0 90 1
		gun 0 -99 "Inhibitor Cannon"
		gun -12 -72 "Inhibitor Cannon"
		gun 12 -72 "Inhibitor Cannon"
		gun -18 -49
		gun 18 -49
		turret 0 2 "Point Defense Turret"
		leak "remnant leak" 30 4
		leak "remnant leak sparkle" 30 10
		explode "tiny explosion" 40
		explode "small explosion" 20
		explode "medium explosion" 10
		"final explode" "final explosion small" 1
		description "One of the strangest and most useful things that the Remnant have discovered in the abandoned worlds of the Ember Waste is alien cloaking technology. They have designed their scout ships to cloak when needed in order to explore alien territory without being seen... or to escape in a hurry after stealing something particularly valuable from those aliens."
		description "	The cloaking technology is built into the Starling's hull; it cannot be removed or transferred to another ship."

mission "Remnant: Crystal Research 5"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Keystone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"

> 	npc "scan outfits" "save"

Reviewer 6: You should probably add a dialog that appears when you scan the ship, like what the FW recon does.

Author: Done. Thanks for the suggestion!

		government "Remnant"

> 		personality coward disables plunders staying surveillance target mining

Author: I gave it those personalities so that it would act effectively like other Remnant - except that it would run away if it got too badly damaged. This is not the same as uninterested, which is fairly bland and basically just translates "ignores the player."

			choice
				`	"Wait, you are a pilot?"`
				`	"Okay."`
					accept
			`	"Oh, yes. It is actually quite common among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a part of a general education. It is especially worthwhile for those of us who work directly with starships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures suggest both pride and humility as he talks about his piloting skills. "Anyway, let's go!"`
				accept
	on visit
		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000

mission "Remnant: Crystal Research 6"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Key Stone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"

> 		personality coward disables plunders staying surveillance target mining

Reviewer 6: ```suggestion
		personality staying target uninterested
```

Author: I gave it those personalities so that it would act effectively like other Remnant - except that it would run away if it got too badly damaged. This is not the same as uninterested, which is fairly bland and basically just translates "ignores the player."

		system "Peragenor"
		ship "Starling" "Winds of Light"
	to offer
		has "Remnant: Crystal Research 5: done"
	on offer
		conversation
			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a Key Stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear and watches as the two ships head back out of the dome.`
				launch
	on visit
		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation

> 			`The scanning has proceeded smoothly. While in transit, you took the opportunity to examine the preliminary results: a few differences between the two crystals, but both appear to have functioned similarly. Upon landing on Viminal, Aeolus comes over to collect a copy of the data for the researchers.`

Reviewer 6: ```suggestion
			`While in transit, you took the opportunity to examine the preliminary results of the scans: a few differences between the two crystals, but both appear to have functioned similarly. Upon landing on Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
```

> 			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully: "Perhaps more importantly, it may help us understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aid."`

Reviewer 6: ```suggestion
			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully. "Perhaps more importantly, it may help us understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aid."`
```

					goto foodvolume
				`	"Maybe I should import some spices."`
					goto spices
				`	"What is our next step?"`
					goto deliverdata
			`	Aeolus chuckles and makes an amused gesture. "An army lives and dies by its food! A poorly fed soldier is but a short step away from being a deserter, or even a traitor. While we have had lean years in the past, these days we ensure everyone is well fed."`
				goto deliverdata
			label foodvolume
			branch capitallicense
				"license: Remnant Capital" = 1
			`	He looks at you appraisingly. "You are perceptive, but there are some things you cannot learn yet. Ask us again when you have earned more of our trust."`
				goto deliverdata
			label capitallicense
			apply "remnant symbionts info" += 1
			`	He looks at you appraisingly "You are perceptive, and have earned our trust." He leans forward and holds up his hands. He pokes at the leather fingerless gloves he is wearing, and you realize that they extend up under his sleeves. You note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`

> 			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones," He explains while holding up his hands, "Have a knack for accuracy and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed through us and require regular care and attention. Normally, this causes us no hardship; but if we do not eat enough we will consume our reserves far faster than we normally would."`

Reviewer 6: ```suggestion
			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones," He explains while holding up his hands, "have a knack for accuracy and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed through us and require regular care and attention. Normally, this causes us no hardship, but if we do not eat enough, we will consume our reserves far faster than we normally would."`
```

				goto deliverdata
			label creepy
			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts to being useful leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`
			label deliverdata
			`	He pulls out a small metal briefcase and sets it on the table. "Another delivery mission for you, Captain. A short one, at that. We need you to take this case of data crystals with our preliminary data to some researchers on <destination>. This will allow them to combine the data we have collected with their most recent findings, and perhaps help us take a step forward towards understanding the nature of both these crystals and the wormholes. Will you do it?"`
			choice
				`	"Okay."`
					accept
				`	"Not right now."`
					defer
	on complete
		payment 100000
		conversation

> 			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain." she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them...." she makes a convoluted gesture that you think translates to "knowing that you don't know."`

Reviewer 6: ```suggestion
			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain." she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them..." she makes a convoluted gesture that you think translates to "knowing that you don't know."`
```

			label end
			`	"All right, enough history." He turns to the technicians who are finishing the assembly. "The tentacle has been treated to resist decay and the stock mechanisms will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
			scene "outfit/void rifle"

mission "Remnant: Crystal Research 1"
	name "Retrieve more Crystals"

> 	description `Retrieve Crystals from Greenwater, and then Allhome.`

Author: Changed. Lower-cased all the crystals, expanded the description, and swapped in a text replacement for the destination.

	minor
	source "Viminal"
	destination "Greenwater"

> 	cargo "(reserved)" 2

Author: Changed to reserve the full 19.

	to offer
		random < 30
		has "Remnant: Learn Sign 2: done"
		has "Remnant: Technology Available: offered"
	on offer
		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`
		conversation
			`In the spaceport cafeteria, you spot the outfitter who asked you to retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`
			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "You came in here with the appearance of looking for something. Is there something on your mind?"`
			choice
				`	(Ask how business is going.)`

> 				`	(Ask if there are more jobs like the last one you did for him.)`

Author: OK. Sounds good.

					goto crystaljobs
			`	"Well, fairly well. Stocks are up, there are quite a few new outfits in the current, and some recent supply line disruptions have been resolved." He gestures appreciatively. "I am told that we have you to thank for some of that.`

> 			`	"Those crystals you brought back have been quite interesting. The process of dismantling them for analysis yielded some interesting results." He notices your expression. "Oh, you were expecting me to sell them, weren't you?"`

Author: Replaced second "interesting" with "intriguing".

			choice
				`	(Nod affirmatively.)`
					goto notforsale
				`	(Shrug noncommittally.)`
					goto notforsale

> 				`	"No, there are better uses for them."`

Author: OK. Switched it to `"No, I suspected they would be studied instead."`

					goto notreally

> 				`	"What does 'outfits in the current' mean?"`

Author: I'd prefer to leave it as is, then. To me, this seems like the point in a conversation that could naturally have either a conversational or a non-conversational response. This is convey with the action / dialog format with the first two being non-conversational actions.

				goto clarification
			label notforsale
			`	He looks slightly embarrassed. "Sorry to disappoint you, but none of those outfits were ever going to be used in the form they came in, for several different reasons." He ticks them off on his fingers as he lists them. "First, security: Slapping an alien device on every ship without having thoroughly dismantled them would be an invitation to getting tracked. Second, research: Something that does what the Quantum Key Stone does without being identical to it? Definitely interesting to our researchers. And third, purpose."`
			`	He pauses to take a drink. "Unlike what you are probably used to from the outfitters you dealt with back in the ancestral worlds, my purpose is not to make money. Instead, I ensure that Remnant captains have the resources they need. What you see listed as the 'price', Remnant captains see listed instead as a 'logistics adjustment' that indicates the amount of resources are needed to supply that component. Every ship has a resource budget allocated to it, determining how much is can utilize for repairs, restocking, and improvements."`
			label clarification
			choice
				`	"Wait, you didn't sell any of that?"`

> 				`	"Okay, so that shipment you had me get?"`

Author: Fixed, thanks!

					goto crystaljobs

> 			label crystaljobs

Author: Fixed.

> 			`	"I have to apologize for that. We knew from our histories that anyone from the ancestral worlds would be attracted by the opportunity to earn a profit, and we needed to give you both a reason to stay, and a way to gauge your willingness to help. Your shipment was definitely useful, but we had to ensure we framed your job in a way that made sense without revealing what we were actually doing.`

Author: I have added the option for the player to ask why Aeolus would assume that they needed a financial motivation.

			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
			choice
				`	(Sign in the affirmative.)`

> 				`	(Decline.)`

Author: Even if signing is the same as talking for the Remnant, it isn't the case for <first>. Signing the affirmative, like nodding "yes", is an action, and doesn't feel right to me. 

That being said, I've added a "Yes." dialog option here just to give players the choice. Do you want to nod affirmatively? (sort of a casual response) or do you want to boldly declared "Yes."? There is no difference in the conversation, just gives the player a different way of stating the same thing.

If I have learned nothing else from playing Mass Effect, it is that having multiple ways of saying the same thing can lead to me personalizing my character in vastly different ways. Casually nodding affirmatively implies a different way of acting than declaring "Yes." does. Even if the end result is the same, it is how I got there that is important.

			`	Aeolus looks faintly amused. "While you are certainly adventurous, you are not the only one to have traveled widely. We mapped the Hai systems long ago with cloaked ships, but never landed or contacted any of them. So we never knew anything more than what we could observe from orbit or from scanning ships. But you are known to them, and have a fairly public reputation. By using your transponder codes, we can place and pay for orders through hyperspace relays. All we need is a ship in position to feed the signal into the network.`
				accept
			label remnantuntrusted
			`	Aeolus smiles mysteriously at you. "Let's just say that you are not the only resourceful person working for us. We will share more with you once you have demonstrated your loyalty."`
				accept
	on complete
		dialog
			`It takes a few minutes to contact the yardmaster, but once you do, a mixed crew of Hai and Human dockworkers have the crates moved promptly from the warehouse to your cargo bay. Hai and Humans seem accustomed to working together here.`

mission "Remnant: Crystal Research 2"
	name "Retrieve more Crystals"

> 	description `Retrieve Crystals from Greenwater, and then Allhome.`

Author: Because the player *may* be arriving via JD instead of through the wormhole; and it is likely the Remnant set it up from a more holistic view as a sweep of stops across the area instead of going by what is easiest for an HD.

	landing
	source "Allhome"
	destination "Newhome"
	stopover "Makerplace"
	stopover "Stonebreak"
	stopover "Giverstone"
	stopover "Cloudfire"
	stopover "Snowfeather"
	stopover "Icelake"
	stopover "Heartvalley"
	cargo "Keystones" 4
	to offer
		has "Remnant: Crystal Research 2: done"
	on offer
		conversation
			`True to Aeolus' word, there are crates of Hai Quantum Keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message has arrived. It has no indication of provenance, but appears to be a notification that a payment of 250,000 marked 'on-assignment allowance' has been made to your account.`
			`	The attached message is brief and to the point. "Greetings, Captain <first>. Hopefully the shipments were waiting for you. We have made arrangements for samples to be acquired from Makerplace, Stonebreak, Giverstone, Cloudfire, Snowfeather, Icelake, and Heartvalley. Once you have collected all those, head to Newhome. There are a few other places that we are still working to set up, and we should have those ready by the time you reach Newhome."`
				accept
	on stopover
		dialog

> 			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to provide the smoothest flight for your passengers!" reads one of the labels.`

Author: Every appliance or house fixture I've ever owned has come with brochures advertising more of whatever it was I bought and talking about how good a purchase it was. I mean, really. It is a stove. If I haven't managed to recycle this brochure by the time this one needs to be replaced, then chances are I'm not going to trust the same brand a second time. And yet, it still happens. Maybe they're hoping that I'll have such a good experience with the new stove that I give the brochure to my friends?

I included it because *I* notice things like whether or not brands are listed on the package, so I assume that my pilot will, too. I based a fair bit of this on the description of the keystone, which states:

> "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."

To me, that sounds like prime grounds for advertising that this or that brand makes a better version. Either that, or there is a monopolistic single supplier whose product over the millenia has reached religious-symbol level of devotion. And while we haven't really seen much or anything of Hai spiritual beliefs, they do have strong "Just like humanity, except better" traits. Hence the advertising descriptions are intended to sound like they are targeted at either star liner captains (as in, captains that cater to rich/upper class passengers), or well-off people themselves. Given what we know of the Hai and the seemingly intended "just like humanity but better" feel, having a well-functioning capitalistic society with a whole variety of of suppliers makes sense. 

Worth noting that - Remnant attitudes aside - the price of a Keystone versus the price of a Key Stone indicates that the Remnant superiority might not be as clearly superior as they would like to think.

	on complete
		payment 250000
		dialog
			`You are just checking in with the port authorities to locate your shipment when your commlink indicates a pair of messages incoming. While you wait for the second one to load, you see the first message is a deposit statement indicating that a payment of <payment> has been deposited from a "Solar Winds Logistics Ltd."`

mission "Remnant: Crystal Research 3A"
	invisible
	landing
	source "Makerplace"
	destination "Newhome"

> 	cargo "Keystones" 2

Author: Thanks for the heads-up.

If I reserve all the space at the start of the mission, wouldn't that mean that by the end of it the player would have twice the space taken up (half by the "reserved" and half by the actual keystones?

> 	cargo "Keystones" 2

Author: OK. As suggested, I've got all the missions set up so that it will appear on the first planet, and the rest will happen in sequence. I'm fine with the possibility of a slight desync.

	to offer
		has "Remnant: Crystal Research 3: active"
	on offer
		conversation
			`Sure enough, there is a pair of crates waiting for you. This one depicts of a pair of distinguished elderly Hai relaxing in front of a matte painting.`

> 				accept

Author: Only one planet has a conversation. Things have been changed so that it will happen in sequence.

	landing
	source "Stonebreak"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3C"
	invisible
	landing
	source "Giverstone"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3D"
	invisible
	landing
	source "Cloudfire"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3E"
	invisible
	landing
	source "Snowfeather"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3F"
	invisible
	landing
	source "Icelake"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 3G"
	invisible
	landing
	source "Heartvalley"
	destination "Newhome"
	cargo "Keystones" 2
	to offer
		has "Remnant: Crystal Research 3: active"

mission "Remnant: Crystal Research 4"
	name "Retrieve more Crystals"
	description `Return the Crystals to Viminal.`
	landing
	source "Newhome"
	destination "Viminal"

> 	cargo "Keystones" 19

Author: Blocked message added.

	to offer
		has "Remnant: Crystal Research 3: done"

> 	on offer

Author: Switched to being an `on accept`

		dialog

> 			`As you were reading the previous message, a freight hauler arrives with the last few crates and starts unloading them. Looking over the assembled stack, you note that each crate bears the logos and advertisements for a different organization, shipper, or enterprise. The second message turns out to be a confirmation that you have picked up 19 Keystones. While there are no new instructions, the subtext is clear: Time to head back.`

Author: Moved to an "On Accept".

	on complete
		payment 2000000
		conversation

> 			`As you descend through the clouds to Viminal, you get the feeling someone is watching you, but your sensors show nothing more than a smattering of Gulls and a patrol squad in the distance. Looking down, you spot an extra battery of anti-air artillery next to the main entrance to the spaceport. You note the artillery tracking you and quickly tap in the recognition code to the IFF system. Within seconds, the large guns are swinging back to covering the initial approach.`

Author: I think this is a better pattern. There *is* a scene break there, between the scene on the ship and the on landing text which takes place outside the ship. 

While I consider this to *be* one of the earlier mentions of IFF that can be built off in the future, I'd be happy to add an earlier mention if necessary.

			choice
				`	"Wait, you are a pilot?"`
				`	"Okay."`
					accept
			`	"Oh, yes. It is actually quite common among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a part of a general education. It is especially worthwhile for those of us who work directly with starships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures suggest both pride and humility as he talks about his piloting skills. "Anyway, let's go!"`
				accept
	on visit
		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000

mission "Remnant: Crystal Research 6"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the Key Stone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"

> 	npc "scan outfits" "save"

Author: Fixed. Thanks!

		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling" "Winds of Light"
		dialog
			`The scanner chimes softly once again, informing you that it has completed scanning the Winds of Light and has successfully archived the data from this second scanning.`
	to offer
		has "Remnant: Crystal Research 5: done"
	on offer
		conversation
			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a Key Stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the Keystone to a lift and disconnects it from the ship. As the Keystone is hauled up to the roof another lift lowers the replacement Key Stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`

> 			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. The worker nimbly steps clear and watches as the two ships head back out of the dome.`

Author: Changed. Thanks!

				launch
	on visit
		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation
			`While in transit, you took the opportunity to examine the preliminary results of the scans: a few differences between the two crystals, but both appear to have functioned similarly. Upon landing on Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully. "Perhaps more importantly, it may help us understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aid."`

> 			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`

Author: The term is used earlier in `"Remnant: Expanded Horizons Quarg 2"`. 

While I agree that it could be better to have it introduced, I also suspect that <first> <last> is sufficiently bright to realize a data holding device whatever the form. Case in point, data storage today. 5.25 floppy, 3.5 floppy, CD, DVD, zip drive, omega drive, external hard drive, and USB drives in hundreds of various form factors, including partial connections that don't even make a full USB. I doubt the future has any fewer data storage devices, so I don't see any problem with <first> recognizing a data storage device even if it looks kind of odd.

mission "Remnant: Crystal Research 7"
	name "Deliver Data to <destination>"
	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
	source "Viminal"
	destination
		government "Remnant"
		not planet "Viminal"
	to offer
		has "Remnant: Crystal Research 6: done"
	on offer
		conversation
			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have become accustomed to a few of the dishes, most of them are still very unappetizing.`
			choice
				`	"The Remnant sure like to eat."`
				`	"Why do the Remnant eat so much?"`
					goto foodvolume
				`	"Maybe I should import some spices."`
					goto spices
				`	"What is our next step?"`

> 					goto deliverdata

Author: OK, there are now a couple of mentions of large amounts of food in earlier missions. Likewise, the news entries introduced in #5209 include a `remnant cafeteria` set that all mention large amounts of food.

			`	Aeolus chuckles and makes an amused gesture. "An army lives and dies by its food! A poorly fed soldier is but a short step away from being a deserter, or even a traitor. While we have had lean years in the past, these days we ensure everyone is well fed."`
				goto deliverdata
			label foodvolume
			branch capitallicense

> 				"license: Remnant Capital" = 1

Reviewer 7: Remember: = is for assignment, == is for comparison, so this should be == 1. But really you should use `has`.
```suggestion
				has "license: Remnant Capital"
```

			`	He looks at you appraisingly. "You are perceptive, but there are some things you cannot learn yet. Ask us again when you have earned more of our trust."`
				goto deliverdata
			label capitallicense
			apply "remnant symbionts info" += 1
			`	He looks at you appraisingly "You are perceptive, and have earned our trust." He leans forward and holds up his hands. He pokes at the leather fingerless gloves he is wearing, and you realize that they extend up under his sleeves. You note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`

> 			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones," He explains while holding up his hands, "have a knack for accuracy and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed through us and require regular care and attention. Normally, this causes us no hardship, but if we do not eat enough, we will consume our reserves far faster than we normally would."`

Reviewer 7: Ways to highlight this idea before this point:
* Mentioned the gloves as being something that all Remnant wear earlier on. E.g. have it be a part of their description in the very first mission. (Perhaps if the player refuses the blood test but doesn't run and then gets grabbed by somebody, there could be a mention of the texture of the glove. Perhaps the player notices the pulsating when touching the "glove.")
* There are various feats of great strength or agility that the Remnant show in other missions, though they aren't very directly highlighted, only being said to have occurred. Perhaps have the player react with surprise/shock to one such past event.
* We know the Remnant have camels. Perhaps in a previous mission or in the description of a planet/for a spaceport, mention something about these camels perhaps having some sort of symbiote on them, or simply mention some odd "growth" on them.

Author: Firstly, not all Remnant have the glove symbiont, although probably quite a few people in military or trades specialties do.

Secondly, this *is* supposed to be the early mission that gives the player the first hints. In the grand scheme of things, unless the player is very unlucky, this set of missions will be completed during the early part of the "Rising action" phase of the Remnant story. While veteran players with old pilots will encounter this mission after having acquired the capital license; most future pilots will encounter this before they have done the Salvage mission chain. (Note that the initial Remnant Crystal research mission requires "Remnant: Learn Sign 2: Done", whereas Salvage 1 requires "Remnant: Learn Sign Follow Up: Offered" So at the very least, the player has a 60 day window in which they might get the crystal research mission before they could possibly get the first salvage mission. As such, most players will get a teasing hint that yes, the fact that they eat so much is actually noteworthy and possibly important, but not something the Remnant will trust them with yet.)

Third, I added a bit to the Remnant first contact mission, as well as one of the salvage missions. The former is a note regarding quantity of food; and the later is noting that some of the Remnant the player is evacuating are meditating while an IV feeds into their glove. I suspect most people will assume it feeds through their glove into the person, but perceptive people might be curious as to whether it is literally actually feeding the glove.

I'm currently working on expanding something to do with one or more of the camels.

			label terribleflavor
			`	"I suppose it might. Our symbionts have different dietary requirements, and such unusual compounds in our food inevitably change the flavor. We are all long since used to it, but it would definitely be different for an outsider." He seems amused by this. "But enough of food - there is work to do!"`
				goto deliverdata
			label creepy
			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts to being useful leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`
			label deliverdata
			`	He pulls out a small metal briefcase and sets it on the table. "Another delivery mission for you, Captain. A short one, at that. We need you to take this case of data crystals with our preliminary data to some researchers on <destination>. This will allow them to combine the data we have collected with their most recent findings, and perhaps help us take a step forward towards understanding the nature of both these crystals and the wormholes. Will you do it?"`
			choice
				`	"Okay."`
					accept
				`	"Not right now."`

> 					defer

Reviewer 7: If the player defers then they'll go through this entire conversation again. Should probably just force an accept since the player's gone this far.

Author: Makes sense. Changed.

	on complete
		payment 100000
		conversation

> 			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain." she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them..." she makes a convoluted gesture that you think translates to "knowing that you don't know."`

Reviewer 7: ```suggestion
			`A young researcher is waiting for you when you exit the <ship>. "Thank you for your assistance, Captain," she signs with one hand as she accepts the briefcase with the other. "There has been a lot of concentrated study on the wormholes over the past several centuries. While we know much about them..." she makes a convoluted gesture that you think translates to "knowing that you don't know."`
```

			`	"Now, thanks to your acquisitions, we have the opportunity for new insights. We are looking forward to learning more about these anomalies. May the Embers burn bright for you, Captain." With that, she turns and heads off towards a lab.`

> 			`	As you contemplate the implications of the Remnant studying the wormholes, your commlink chimes with a notification of another payment being made to your account.`

Reviewer 7: ```suggestion
			`	As you contemplate the implications of the Remnant studying the wormholes, your commlink chimes with a notification of another payment of <payment> being made to your account.`
```

		has "Remnant: Learn Sign 2: done"
		has "Remnant: Technology Available: offered"
	on offer
		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`
		conversation
			`In the spaceport cafeteria you spot the outfitter who had you retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`

> 			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "you came in here looking like you were watching for something. Anything on your mind?"`

Reviewer 3: `ventures, "you came in here`
"You" should be capitalized, I believe.

			choice
				`	"Wait, you didn't sell any of that?"`
				`	"OK, so that shipment you had me get?"`
					goto crystaljobs
			`	"You are correct. Despite what was said, none of it was sold - all of it was dismantled by researchers examining how it was made and what its properties are." He pauses to finish one plate before continuing.`
			label crystaljobs
			`	"I have to apologize for that. We knew from our histories that anyone from the ancestral worlds would be attracted by the opportunity to earn a profit, and we needed to give you both a reason to stay, and a way to guage your willingness to help. Your shipment was definitely useful, but we had to ensure we framed your job in a way that made sense without revealing what we were actually doing.`
			`	"So no, I don't have any jobs 'just like the last one,' but I know our researchers would like another shipment. Are you interested?"`
			choice
				`	(Sign the affirmative.)`
				`	(Decline.)`
					decline

> 			`	"Great. They would like you to pick up another 19 keystones. Two differences this time: First, we would like you to pick them up from several different planets. And second, by the time you reach Hai space we will have made arrangements so the shipments will be already paid for and waiting for you. Pick up the first stones on Allhome, then the next set on Greenwater. I will have a list sent to you by the time you reach Greenwater."`

Reviewer 3: Capitalize `keystones`. Looks like I missed this the first time.

			`	As you finish your own shutdown routine, your commlink informs you that your account has been updated with <payment>.`

ship "Starling" "Starling (Keystone)"
	outfits
		"Inhibitor Cannon" 3
		"Point Defense Turret"
		
		"Epoch Cell"
		"Crystal Capacitor"
		"Emergency Ramscoop" 2
		"Quantum Keystone"
		"Salvage Scanner"
		
		"Forge-Class Thruster"
		"Forge-Class Steering"
		"Crucible-Class Steering"
		"Hyperdrive"

mission "Remnant: Crystal Research 5"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the keystone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	to offer
		has "Remnant: Crystal Research 4: done"
	on offer
		conversation
			`You exit the ship to find that the pilot is actually Aeolus, who has already marshaled some dock workers with a number of wagons and lifters to unload the cargo.`

> 			`	"Ah, <first>, thank you for retrieving all these keystones." He sweeps some kind of a device over the crates, then opens one. He pokes at the contents, and pulls out a brochure with a picture of a quantum keystone and a pair of respectable looking Hai. "What's this?" He asks curiously. "Because I bought a keystone, they think I might need more? Our histories told us that humanity thrived on the pursuit of wealth, it just seems inefficient to me."`

Author: Changed the last sentence from 

> "Because I bought a keystone, they think I might need more? Our histories told us that humanity thrived on the pursuit of wealth, it just seems inefficient to me."

to

> "Because I bought a keystone, they think I might need more? That seems like a decidedly inefficient way of promoting their product."

More of a direct and relevant statement, less philosophizing about humanity's economics. Which actually aren't even really relevant considering it is a *Hai* source, not a *Human* source.

> 			`	"Ah, <first>, thank you for retrieving all these keystones." He sweeps some kind of a device over the crates, then opens one. He pokes at the contents, and pulls out a brochure with a picture of a quantum keystone and a pair of respectable looking Hai. "What's this?" He asks curiously. "Because I bought a keystone, they think I might need more? Our histories told us that humanity thrived on the pursuit of wealth, it just seems inefficient to me."`

Reviewer 5: Seems good.

			choice
				`	"I'm not a fan of it either."`
				`	"Those 'inefficient' capitalists did provide a keystone at one twentieth the cost you do."`
					goto capitalists
			`	"You will probably do well here, then. Keep production and resources focused on the goal, that always made the most sense to me." Aeolus signs with confidence. "But economic theory aside, we have some science to do.`
				goto work
			label capitalists
			`	"You've got a point there." Signs Aeolus. "Assuming they value a credit the same as we do and they aren't taking a loss, then they do have a significant production and processing cost advantage." He takes on a defensive tone. "But there is an entire supply chain at work. For all we know, most of these crystals could be simply recycled from older ships. And that is something we want to figure out." He brightens again "That being said, we already have the first steps set out."`
			label work

> 			`	"Our researchers have decided that we need to test how a ship performs with one of these keystones compared to a regular key stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the quantum keystone built with Hai crystals for a quantum key stone made from our crystals. Then we will go back to Peragenor and perform another scan. By using the same ships for both tests, we should be able to get the best data."`

Reviewer 5: ```suggestion
			`	"Our researchers have decided that we need to test how a ship performs with one of these keystones compared to a regular keystone. My ship has been outfitted with one built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the quantum keystone built with Hai crystals for one made from our crystals. Then we will go back to Peragenor and perform another scan. By using the same ships for both tests, we should be able to get the best data."`
```

				accept
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling (Keystone)" "Winds of Light"
		dialog
			`The scanner chimes softly, informing you that it has completed scanning the Winds of Light and has successfully archived the data.`
	on visit
		dialog `You land on <planet>, but you haven't scanned the Starling in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000

mission "Remnant: Crystal Research 6"
	name "Scan the <npc> in <waypoints>"
	description `Go scan the <npc> to record the key stone post-wormhole.`
	landing
	waypoint "Peragenor"
	source "Viminal"
	destination "Viminal"
	npc "scan outfits" "save"
		government "Remnant"
		personality coward disables plunders staying surveillance target mining
		system "Peragenor"
		ship "Starling" "Winds of Light"
		dialog
			`The scanner chimes softly once again, informing you that it has completed scanning the Winds of Light and has successfully archived the data from this second scanning.`
	to offer
		has "Remnant: Crystal Research 5: done"
	on offer
		conversation

> 			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a key stone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the keystone to a lift and disconnects it from the ship. As the keystone is hauled up to the roof another lift lowers the replacement key stone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`

Reviewer 5: ```suggestion
			`When you and the <npc> settle back onto the landing pad, there is already a dockworker waiting with a keystone. Once the Starling is settled, the dockworker climbs up the outside of the ship, hooks the keystone to a lift and disconnects it from the ship; another lift soon lowers the replacement keystone. The dockworker quickly installs it and jumps down, landing gracefully from the several-meter drop.`
```

Author: In this scene there is actually both a keystone and a key stone present. The keystone is being removed and lifted off, while the replacement key stone is being lowered into place.

Author: Also, in terms of replacing the "`As the keystone is hauled up to the roof another lift lowers the replacement`" with "`another lift soon lowers the replacement`", the idea was to subtly convey the idea that the Remnant have their shipyards set up for fast turn around. The Remnant don't have much "defense in depth" to give them time to repair or refit ships. They need to be able to scramble and get ships fixed up and shipped out as fast as they possibly can; and part of that means that their rigging is setup so that replacement parts are already arriving while the old parts are still in the process of being removed. In other words, simultaneous operations.

I didn't want to *force* player realization of this fact, but I do want the details to be present so that they *can* notice it.

Reviewer 5: I think the distinction between a "keystone" and a "keystone" is likely to be lost on most players and considered a typo--as I did. :)

OK on the other point.

> 			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. You check your cameras and see the worker nimbly steps clear of the wake as the two ships head back out of the dome.`

Reviewer 5: ```suggestion
			`	"Ready? Let's go!" Aeolus signs as he fires up his systems and gestures an acknowledgement to the dockworker. You check your cameras and see the worker nimbly step clear of the engine exhausts as the two ships head back out of the dome.`
```

				launch
	on visit
		dialog `You land on <planet>, but you haven't scanned the <npc> in <waypoints>. Make sure you scan the ship before returning.`
	on complete
		payment 150000
		conversation
			`While in transit, you took the opportunity to examine the preliminary results of the scans: there are a few differences between the two crystals, but both appear to have functioned similarly. Upon landing on Viminal, Aeolus comes over to collect a copy of the data for the researchers.`
			`	"Thanks for your help, Captain <first>. This information may be valuable in understanding how the wormholes work." He pauses, then continues thoughtfully. "Perhaps more importantly, it may help us understand how the wormholes of the Ember Waste are different from the wormholes elsewhere that require no such aid."`
			`	He looks down at the data crystal. "I have one more task to ask of you. Meet me in the cafeteria in an hour to discuss it. May the Embers burn bright for you, Captain." He gives a wave that seems vaguely like a salute and heads off.`

> 			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have combined them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`

Reviewer 5: ```suggestion
			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so whichever Remnant person or agency is sending you these payments must have combined them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`
```

	name "Deliver Data to <destination>"
	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
	source "Viminal"
	destination
		government "Remnant"
		not planet "Viminal"
	to offer
		has "Remnant: Crystal Research 6: done"
	on offer
		conversation
			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have become accustomed to a few of the dishes, most of them are still very unappetizing.`
			choice
				`	"The Remnant sure like to eat."`
				`	"Why do the Remnant eat so much?"`
					goto foodvolume
				`	"Maybe I should import some spices."`
					goto spices
				`	"What is our next step?"`
					goto deliverdata
			`	Aeolus chuckles and makes an amused gesture. "An army lives and dies by its food! A poorly fed soldier is but a short step away from being a deserter, or even a traitor. While we have had lean years in the past, these days we ensure everyone is well fed."`
				goto deliverdata
			label foodvolume
			branch capitallicense
				has "license: Remnant Capital"

> 			`	He looks at you appraisingly. "You are perceptive, but there are some things you cannot learn yet. Ask us again when you have earned more of our trust."`

Reviewer 5: ```suggestion
			`	He looks at you appraisingly. "There are some things you cannot learn yet. Ask us again when you have earned more of our trust."`
```

				goto deliverdata
			label capitallicense
			apply
				"remnant symbionts info" += 1

> 			`	He looks at you appraisingly "You are perceptive, and have earned our trust." He leans forward and holds up his hands. He pokes at the leather fingerless gloves he is wearing, and you realize that they extend up under his sleeves. You note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`

Reviewer 5: ```suggestion
			`	He leans forward and holds up his hands, poking at the leather fingerless gloves he is wearing, and you realize that they extend up under his sleeves. You note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`
```

> 			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones," He explains while holding up his hands, "have a knack for accuracy and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed through us and require regular care and attention. Normally, this causes us no hardship, but if we do not eat enough, we will consume our reserves far faster than we normally would."`

Reviewer 5: ```suggestion
			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice it to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones," He explains while holding up his hands, "have a knack for accuracy and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed through us and require regular care and attention. Normally, this causes us no hardship, but if we do not eat enough, we will consume our reserves far faster than we normally would."`
```

			choice
				`	"That is quite the revelation."`
				`	"Thanks for the explanation."`
				`	"Is that why some of the food tastes terrible?"`
					goto terribleflavor
				`	"Sounds creepy."`
					goto creepy

> 			`	Aeolus nods. "From what I have heard, humanity does not even seem to have much in the way of exoskeletons or cybernetics. Having living organisms we wear would certainly seem to be the realm of science-fiction." He contemplates his symbiont for a few seconds as he absent-mindedly eats more of his meal. "But education aside, there is work to do."`

Reviewer 5: ```suggestion
			`	Aeolus nods. "From what I have heard, the ancestral worlds do not seem to have much in the way of exoskeletons or cybernetics. The living organisms we wear would certainly seem to be the realm of science-fiction." He contemplates his symbiont for a few seconds as he absent-mindedly eats more of his meal. "But education aside, there is work to do."`
```

				goto deliverdata
			label spices
			`	He shakes his head. "I will not discourage you doing so - and I am sure some might appreciate them - but most of us like our food the way it is." He chuckles, then continues. "But there is still work to do."`
				goto deliverdata
			label terribleflavor
			`	"I suppose it might. Our symbionts have different dietary requirements, and such unusual compounds in our food inevitably change the flavor. We are all long since used to it, but it would definitely be different for an outsider." He seems amused by this. "But enough of food - there is work to do!"`
				goto deliverdata
			label creepy

> 			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts to being useful leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`

Reviewer 5: ```suggestion
			`	"Creepy? I do not understand how something so useful and beneficial could be 'creepy,' but I suppose some might equate our symbionts with leeches. To each their own, I suppose." He shrugs, then continues. "But enough. We still have work to do."`
```

	on complete
		payment 250000
		dialog

> 			`True to Aeolus' word, there are crates of Hai quantum keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message from Viminal has arrived. It is a notification that a payment of <payment> has been made to your account, with a note attached that simply states "on-assignment allowance."`

Reviewer 3: Capitalize `quantum keystones`.

		conversation
			`You exit the ship to find that the pilot is actually Aeolus, and he has already marshalled a few dock workers with a number of wagons and lifters to unload the cargo.`
			`	"Ah, <first>, thank you for retrieving all these Keystones." he pauses to stare at a crate bearing the image of a bunch of smiling Hai, then shakes his head and turns back to you.`
			`	"Our researchers have decided that we need to test how a ship performs with one of these Keystones compared to a regular Key Stone. My ship has been outfitted with a keystone built from the samples you brought us earlier. We request that you meet us in Peragenor, where you will scan us after going through the wormhole. Then we will return here and swap out the Quantum Keystone built with Hai crystals for a Quantum Key Stone made from our crystals. Then we will go back to Peragenor and get scanned again. By using the same ships for both tests, we should be able to get the best data."`
			choice
				`	"Wait, you are a pilot?"`
				`	"Okay."`
					accept

> 			`	"Oh, yes. It is actually quite a common skill among the Remnant. I am not a particularly good pilot, but it is considered an important skill that is taught to almost everyone as a general part of general education. It is especially worthwhile for those of us who work directly with ships on a daily basis and may be called upon to move or fly one at a moment's notice." His gestures carry tones of both pride and humility as he talks about his piloting skills. "Anyway, let's go"`

Reviewer 3: With this edit, you use `skill` in two concurrent sentences. I would suggest rewording `quite a common skill among` to `quite common among` to avoid this.
`general part of general education`: Definitely remove a `general` here. I'd recommend the first.
